app/assets/javascripts/osem.js

```diff
@@ -147,6 +147,15 @@ function word_count(text, divId, maxcount) {
     });
 };
 
+function fill_if_empty(text_area, filler) {
```

Comment:
There is a clear goal to these function and the name of the function is clear and matches towards the goal of the PR.